csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -154,7 +154,7 @@ Secondly, the program fulfills the following for all items of:
   Sets the value as given in the configuration of the program or the corresponding argument the program was invoked with.
   If values from both sources are present, the program SHOULD prefer the latter one.
   The program SHALL NOT use hard-coded values.
-* `/document/tracking/id`: If the element `cvrf:ID` contains any line breaks or leading or trailing white space,
+* `/document/tracking/id`: If the element `cvrf:ID` contains any newline sequence or leading or trailing white space,
```

Comment:
I am not so sure anymore, if our made up trerm "newline sequence" is as helpful as we think. We want unicode (utf-8) and thus the [Unicode Standard Annex \#14 - UNICODE LINE BREAKING ALGORITHM](https://www.unicode.org/reports/tr14/tr14-32.html) might help find us a better term ... or not 🙈.